src/styles/reset/reset.scss

```diff
@@ -0,0 +1,3 @@
+.page {
```

Comment:
hre you can use tag body, in files like reset.scss or normalize.scss we usually give styleas for tags, not for classes